### PR TITLE
SQLite should reference the filename from the .env file instead of development.sqlite

### PR DESCRIPTION
**Pull request by vincevannoort**

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = {
   sqlite: {
     client: 'sqlite3',
     connection: {
-      filename: Helpers.databasePath('development.sqlite')
+      filename: Helpers.databasePath(`${Env.get('DB_DATABASE', 'adonis')}.sqlite`)
```
Comment (RomainLanz) [inline config/database.js]: I believe we should keep `development` as a default here.

**Comment by vincevannoort** [comment]
Alright! I updated it. :)

**Comment by thetutlage** [comment]
Thanks 😄

**Comment by thetutlage** [comment]
Mind creating a PR for api-only boilerplate too? https://github.com/adonisjs/adonis-api-app/blob/develop/config/database.js

**Pull request merged**